### PR TITLE
correct translation for Android and update some strings for QTClient (translation update is on the way)

### PR DESCRIPTION
**Pull request by lbk2907**

### CODE REVIEW

File: Client/qtTeamTalk/keycomp.ui [hunk 1]
```diff
@@ -35,7 +35,7 @@
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
-         <string>Hold down the keys which should be used as a hot key. Modifier keys like Shift, Ctrl and Alt can be used in combination with other keys</string>
+         <string>Hold down the keys that you want to used as a hot key. Modifier keys like Shift, Ctrl and Alt can be used in combination with other keys</string>
```
Comment (bear101) [inline Client/qtTeamTalk/keycomp.ui:38]: This correction is minimal and will invalidate all existing translations. I would prefer to stay with the original

File: Client/qtTeamTalk/keycomp.ui [hunk 2]
```diff
@@ -69,7 +69,7 @@
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
-         <string>This dialog closes when you have released all keys</string>
+         <string>This dialog will close automaticly when you released all keys</string>
```
Comment (bear101) [inline Client/qtTeamTalk/keycomp.ui:72]: This correction is minimal and will invalidate all existing translations. I would prefer to stay with the original

File: Client/qtTeamTalk/keycompdlg.cpp [hunk 3]
```diff
@@ -220,7 +220,7 @@ void KeyCompDlg::keyReleaseEvent(QKeyEvent* event)
     if(close_dlg && m_hotkey[0] != (INT32)MAC_NO_KEY && m_hotkey[1] == (INT32)MAC_NO_KEY)
     {
         QMessageBox::information(this, tr("Invalid key combination"),
-                                 tr("macOS does not support only modifier keys, "
+                                 tr("macOS does not support only modifier keys. "
```
Comment (bear101) [inline Client/qtTeamTalk/keycompdlg.cpp:223]: Should be comma prior to "i.e."

File: Client/qtTeamTalk/mainwindow.cpp [hunk 4]
```diff
@@ -778,7 +778,7 @@ void MainWindow::loadSettings()
     if((!packdir.exists()) && packset != tr("Default"))
     {
         QMessageBox answer;
-        answer.setText(tr("Your sounds pack %1 don't exist, would you like to use default sounds pack?").arg(packset));
+        answer.setText(tr("The sound pack %1 don't exist. Would you like to use default sound pack?").arg(packset));
```
Comment (bear101) [inline Client/qtTeamTalk/mainwindow.cpp:781]: Should be:
The sound pack %1 does not exist. Would you like to use the default sound pack?

File: Client/qtTeamTalk/mainwindow.cpp [hunk 5]
```diff
@@ -969,7 +969,7 @@ void MainWindow::clienteventConLost()
     if(ttSettings->value(SETTINGS_CONNECTION_RECONNECT, SETTINGS_CONNECTION_RECONNECT_DEFAULT).toBool())
         m_timers[startTimer(5000)] = TIMER_RECONNECT;
 
-    addStatusMsg(STATUSBAR_BYPASS, tr("Connection lost to %1 TCP port %2 UDP port %3")
+    addStatusMsg(STATUSBAR_BYPASS, tr("Connection to %1 TCP port %2 UDP port %3 lost")
```
Comment (bear101) [inline Client/qtTeamTalk/mainwindow.cpp:972]: This correction is minimal and will invalidate all existing translations. I would prefer to stay with the original

File: Client/qtTeamTalk/mainwindow.cpp [hunk 6]
```diff
@@ -1518,7 +1518,7 @@ void MainWindow::clienteventUserRecordMediaFile(int source, const MediaFileInfo&
                      .arg(getDisplayName(user)));
         break;
     case MFS_FINISHED :
-        addStatusMsg(STATUSBAR_BYPASS, tr("Finished audio file %1")
+        addStatusMsg(STATUSBAR_BYPASS, tr("Finished writing audio file %1")
```
Comment (bear101) [inline Client/qtTeamTalk/mainwindow.cpp:1521]: Finished writing to audio file %1

File: Client/qtTeamTalk/mainwindow.ui [hunk 7]
```diff
@@ -1872,7 +1872,7 @@
   </action>
   <action name="actionStoreForMove">
    <property name="text">
-    <string>&amp;Store User(s) for Move</string>
```
Comment (bear101) [inline Client/qtTeamTalk/mainwindow.ui:1875]: "Store" is more correct because the user is added to a list that is emptied later

**Review by bear101 — COMMENTED** [review]
Translations should only be changed if they are incorrect. Changing an existing translation will invalidate all exiting languages.

**Comment by lbk2907** [comment]
I already change the string for each translation file. I will push when I'm
done translating my language.

**Comment by lbk2907** [comment]
Or you need me to push it first with the new string except the language
file that I'm translating?

**Comment by bear101** [comment]
Please revert the changes I've mentioned to the English.ts file and then push again.

**Comment by lbk2907** [comment]
I already push the translation file with the new string changes except the one that I’m working on.

**Comment by lbk2907** [comment]
I just correct the string typo for QTClient in mainwindow.cpp line 6024. I also updated it on each translation file.

**Comment by lbk2907** [comment]
I’ll take a look.

**Comment by lbk2907** [comment]
Done. I already make the changes exactly like you mention before.

**Pull request merged**